devlab/provision/cloudferry.sh

```diff
@@ -56,6 +56,19 @@ if [[ ! -d .ubuntu-venv ]]; then
 else
     echo "CloudFerry venv is already present, skipping"
 fi
+
+echo "Preparing configuration for CloudFerry"
+cp devlab/config.template configuration.ini
+
+while read key value
+do
+    value=($value);
+    value=${value[1]};
+    if [ -n $value ];then
```

Comment:
1. Please replace single brackets (`[`, `]`) with double (`[[`, `]]`)
2. Remove all semicolons, they are not required (except after the `if` statement)